stackslib/src/chainstate/stacks/boot/pox_4_tests.rs

```diff
@@ -230,7 +256,7 @@ fn pox_extend_transition() {
     //  tenures start being tracked.
     let EMPTY_SORTITIONS = 25;
 
-    let (epochs, pox_constants) = make_test_epochs_pox();
+    let (epochs, pox_constants) = make_test_epochs_pox(false);
```

Comment:
I'm curious why we aren't considering a case of this test with `make_test_epochs_pox(true)`?  Same question for those below.  Is it because these tests require more than 123 sortitions?